releasenotes/notes/asm-1-click-activation-with-RCM-6a2431949768b73b.yaml

```
@@ -1,4 +1,4 @@
 ---
 features:
   - |
-    ASM: 1-click activation with RCM.
+    ASM: 1-click activation ability through Remote Configuration Management.
```

As we talk in the meeting (cc @brettlangdon @juanjux):
```suggestion
    ASM: 1-click activation ability through Remote Configuration Management. Currently in beta. Set DD_REMOTE_CONFIGURATION_ENABLED to true to enable it.
```

---

Great, thanks for the catch. 
Just referencing #4400 here as the PR that this comment change is in regards to.